tests/component/test_task.py

```diff
@@ -194,3 +203,10 @@ def test___default_arguments___perform_thrmcpl_lead_offset_nulling_cal___no_erro
     ai_thermocouple_task: nidaqmx.Task,
 ) -> None:
     ai_thermocouple_task.perform_thrmcpl_lead_offset_nulling_cal()
+
+
+def test___is_task_done___task_is_done___returns_true(ai_on_demand_task: nidaqmx.Task):
+    assert ai_on_demand_task.is_task_done()
+    ai_on_demand_task.start()
+    ai_on_demand_task.wait_until_done()
+    assert ai_on_demand_task.is_task_done()
```

Comment:
arrange, act, assert
(it's ok to combine act+assert)
```suggestion
    assert ai_on_demand_task.is_task_done()
    ai_on_demand_task.start()
    ai_on_demand_task.wait_until_done()

    assert ai_on_demand_task.is_task_done()
```

Reply:
Changed